rewards/calc_rewards.py

```diff
@@ -65,6 +72,7 @@ def fetch_all_schedules(chain: str, setts: List[str]):
     setts_with_schedules = []
     for sett in setts:
         schedules = logger.getAllUnlockSchedulesFor(sett).call()
+        sett = Web3.toChecksumAddress(sett)
```

Comment:
should this go above the logger call?

Reply:
Calling this function with lowercase vs checksum has no effect, web3 probably checksums this already. Its still good practice to make sure we checksum first anyway,